server/container/guice/cassandra-guice/pom.xml

```diff
@@ -259,11 +259,6 @@
             <groupId>org.awaitility</groupId>
             <artifactId>awaitility</artifactId>
         </dependency>
-        <dependency>
```

Comment:
Maybe @mbaechler  can explain a bit more about it, but it looks like something he added on purpose...

Reply:
IMO we should rather move this dependency to dropwizard metrics sub module anyway

Reply:
Well, that being said, dropwizard don't seem to be magically using the HdrHistogram back-end,
and something like https://bitbucket.org/marshallpierce/hdrhistogram-metrics-reservoir seems required...

I propose to keet @blackheaven  changes.